### ISSUE TITLE
Adding kalker to F-Droid

### ISSUE THREAD

**Issue by R1yuu** [PaddiM8/kalker#104]
It would be really nice if kalker was available through F-Droid. As far as I know, kalker fulfills almost all criteria to be accepted by F-Droid, except some structural requirement of having Metadata about the project in a certain location. It would be amazing if you could propose adding it.

**Comment by PaddiM8** [comment]
I agree! Added some metadata and created an issue in the f-droid repo.

**Comment by R1yuu** [comment]
Hey, are there any Status updates on this?

**Comment by PaddiM8** [comment]
@R1yuu It's listed as ready in the issue tracker, but I need fix one thing. Then the build process needs to be configured.

**Comment by R1yuu** [comment]
Hello, is this still on going?
